setup.py

```diff
@@ -109,24 +75,26 @@ def read_requirements():
 s_ = setup(
     name='neat',
     version=pversion,
-    scripts=['neat/channels/compilechannels'],
+    scripts=['neat/channels/compilechannels',
+             'neat/tools/simtools/neuron/initneuron'],
```

Comment:
should be removed if we use your new option

Reply:
removed initneuron